posts/2020-11-09-ocp-log-forwarding.adoc

```diff
@@ -320,15 +322,36 @@ metadata:
 spec:
   managementState: "Unmanaged"
 ```
-*Note:* After you set this value to *"Unmanaged"*, any further changes to the _ClusterLogging_ or _LogForwarding_ instances are not automatically detected. To automatically detect changes, you must change the *managementState* field back to *"Managed"*.
 
-Next, set the *MERGE_JSON_LOG* environment variable to *true*.
+Alternatively, you can modify this file on the web console. Ensure you are in the `openshift-logging` project namespace and go to `Administration > Custom Resource Definitions > ClusterLogging > Instances > Instance > YAML`
+
+Next, we'll need to change the  *merge_json_log* attribute of the `kubernetes.var.log.containers.**` filter to *true* in the Fluentd's config map.
+
+```
+[root@ocp ~]# oc edit configmap fluentd
 
+apiVersion: v1
+data:
+...
+  <filter kubernetes.var.log.containers.**>
+    @type parse_json_field
+    merge_json_log 'true'
+    preserve_json_log 'true'
+    json_fields 'log,MESSAGE'
+  </filter>
+...
+```
+
+Finally we need to force restart the Fluentd pods to pick up this new change:
 [source]
 ----
-[root@ocp ~]# oc set env ds/fluentd MERGE_JSON_LOG=true
+oc delete pod --selector logging-infra=fluentd
 ----
 
+Alternatively, you can modify this file on the web console. Ensure you are in the `openshift-logging` project namespace and go to `Workloads > Config Maps > fluentd > YAML`
```

Comment:
should this be at line 344? Would the user still need to restart the Fluentd pods if they make the change through the web console? If not, i think this is ok here but if so, recommend moving it to before the restart at line 345.

Also, as mentioned above, should be explicit about what the user should do if they go w/ the web console:

Alternatively, you can modify this file on the web console. Ensure you are in the `openshift-logging` project namespace and go to `Workloads > Config Maps > fluentd > YAML`.  Change the  value for the *merge_json_log* attribute of the `kubernetes.var.log.containers.**` filter to *true*.

Reply:
fixed